crates/glaredb/src/commands.rs

```diff
@@ -60,7 +60,18 @@ impl RunCommand for LocalArgs {
                         "only one of query or an SQL file can be passed at a time"
                     ))
                 }
-                (Some(file), None) => Some(tokio::fs::read_to_string(file.as_str()).await?),
+                (Some(file), None) => {
+                    if file.to_ascii_lowercase() == "version" {
+                        return Err(anyhow!(
+                            "'version' is not a valid command, did you mean '--version'?"
+                        ));
+                    }
```

Comment:
I was thinking about that too. I don't think we'd want to have too many special cases, but I highly doubt someone using glaredb + sql files will ever have a sql file simply named `version`, so I thought this would be a pretty safe special case. We could match on all `--` flags _(currently only --help and  --version)_. 

such as 
```sh
> ./glaredb local help
Error: 'help' is not a valid command, did you mean '--help'?
```

Reply:
> [...] We could match on all -- flags [...]

I don't know if we need to go that far since I imagine the two most common commands that someone trying out for the first time are likely to be `glaredb help` and `glaredb version`.

Reply:
> I don't know if we need to go that far since I imagine the two most common commands that someone trying out for the first time are likely to be glaredb help and glaredb version.

Agreed.